src/style/global.scss

```diff
@@ -165,17 +165,24 @@ h6, .h6 {
             display: none;
           }
           .navbar-nav {
+            display: block;
             .nav-link {
               font-size: $navbar-link-font-size-collapse;
               line-height: $navbar-link-line-height-collapse;
               padding: $navbar-link-padding-y-collapse $navbar-link-padding-x-collapse;
-              margin-right: $navbar-link-margin-x-collapse;
-              margin-left: $navbar-link-margin-x-collapse;
+              margin-right: $navbar-link-margin-r-collapse;
+              margin-left: $navbar-link-margin-l-collapse;
               &-icon {
                 font-size: $navbar-link-icon-size-collapse;
               }
             }
           }
+          .navbar-brand {
+            font-size: 1rem;
+            .mx-3 {
```

Comment:
`mx-3` is a utility class that applies 1rem left/right margins. If you want to style the icon consider using a new class like the following:
```
.navbar-brand-icon {
  margin: 0 0.5rem;
  width: 1.5rem;
  height: 1.5rem;
  display: inline-block;
  line-height: 0;
  vertical-align: middle;
}
```